test/celeritas/user/DetectorSteps.test.cc

```diff
@@ -255,6 +258,7 @@ TEST_F(DetectorStepsTest, TEST_IF_CELER_DEVICE(device))
     EXPECT_VEC_EQ(host_output.event_id, output.event_id);
     EXPECT_VEC_EQ(host_output.track_step_count, output.track_step_count);
     EXPECT_VEC_EQ(host_output.step_length, output.step_length);
+    EXPECT_VEC_EQ(host_output.weight, output.weight);
```

Comment:
I'm not sure if we've got you access to a GPU-enabled machine yet (it would be great to try using CADES?) but this fails due to missing code in `DetectorSteps.cu`, which is the CUDA implementation corresponding to `DetectorSteps.cc`:
```
289: [ RUN      ] DetectorStepsTest.device
289: info: Creating 1 device streams
289: /home/s3j/Code/celeritas/test/celeritas/user/DetectorSteps.test.cc:261: Failure
289:  Size of: output.weight
289:   Actual: 0
289: Expected: host_output.weight.size()
289: Which is: 180
289: Actual values: {};
289:
289:
289: [  FAILED  ] DetectorStepsTest.device (15 ms)
```